lib/nightwatch-handler.js

```diff
@@ -21,6 +22,7 @@ module.exports = function (options) {
     distPath: distPath,
     indexFileName: indexFileName,
     mocksPort: mocksPort,
+    mocksHttpsPort: mocksHttpsPort,
```

Comment:
@jhudson8 
It will be good to support both as teams use mix and match of http and https.